src/main/java/com/mraof/minestuck/blockentity/IntellibeamLaserstationBlockEntity.java

```diff
@@ -0,0 +1,198 @@
+package com.mraof.minestuck.blockentity;
+
+import com.mraof.minestuck.alchemy.AlchemyHelper;
+import com.mraof.minestuck.block.MSBlocks;
+import com.mraof.minestuck.block.machine.IntellibeamLaserstationBlock;
+import com.mraof.minestuck.item.MSItems;
+import com.mraof.minestuck.util.MSSoundEvents;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.protocol.Packet;
+import net.minecraft.network.protocol.game.ClientGamePacketListener;
+import net.minecraft.network.protocol.game.ClientboundBlockEntityDataPacket;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.Containers;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+
+import static com.mraof.minestuck.alchemy.AlchemyHelper.createEncodedItem;
```

Comment:
I would advice against this specific use of a static import

Reply:
why

Reply:
aactually i see why, nevermind